src/Components/RecsListTable/index.js

```diff
@@ -53,6 +54,8 @@ const RecsListTable = () => {
   const page = filters.offset / filters.limit + 1;
   const [filteredRows, setFilteredRows] = useState([]);
   const [displayedRows, setDisplayedRows] = useState([]);
+  const [recommendations, setRecommendations] = useState(recs);
```

Comment:
Do you really need to keep another array of recs? Why you can't modify `filteredRows` instead?

Reply:
Because I don't understand how it work, how should I change them?